### PR TITLE
Fixed csv download and show metadata link in local and custom DC to use local API path

### PR DESCRIPTION
**Pull request by dwnoble**
Before:
Screenshot 2024-07-10 at 5 17 34 PM

Screenshot 2024-07-10 at 5 17 24 PM

After:
Screenshot 2024-07-10 at 5 16 27 PM
Screenshot 2024-07-10 at 5 16 38 PM

### CODE REVIEW

File: static/js/utils/data_commons_client.ts [hunk 1]
```diff
@@ -31,3 +31,12 @@ export const defaultDataCommonsWebClient = new DataCommonsWebClient({
 export const defaultDataCommonsClient = new DataCommonsClient({
   apiRoot: DEFAULT_CLIENT_API_ROOT,
 });
+
```
Comment (juliawu) [inline static/js/utils/data_commons_client.ts:34]: supernit: Could you add a quick docstring for this exported function? Would be helpful to clarify why we need this function and can't just use `new DataCommonsClient({apiRoot})` directly.

**Review by juliawu — APPROVED** [review]
Thanks for the fix!

File: static/js/utils/data_commons_client.ts (hunk 1 above)
Comment (dwnoble) [reply static/js/utils/data_commons_client.ts:34]: done!

**Review by keyurva — APPROVED** [review]
Thanks Dan!

**Pull request merged**